src/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { BrowserWindow } from 'electron'
+
+export function getWindow(): BrowserWindow {
+  return BrowserWindow.getAllWindows()[0]
+}
+
+export function sendAction(action: string, ...args: unknown[]): void {
```

Comment:
How about naming it more specific like `sendChannelToMainWindow()` to use the same term as in the [doc](https://electronjs.org/docs/api/web-contents#contentssendchannel-arg1-arg2-)?

Reply:
Updated!